test.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -ex
```

Comment:
mmmmmm, I don't get this.

Reply:
-e means 'exit with a fail if any individual command fails', -x means 'show each line as they're written.

Reply:
Sorry I edited the comment, I mean the actual's file goal. Is it to verify the installation went well?

Reply:
Yes, exactly. Right now, it validates that the circle config is valid, and that we didn't accidentally leave any of our placeholders in configuration files. I just tried setting it up to clone FBIA and actually run tests there, but it was turning out to be pretty complex so I'll save that for another day.